src/JdbcUtils.java

```diff
@@ -0,0 +1,45 @@
+import org.postgresql.ds.PGSimpleDataSource;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+
+public class JdbcUtils {
```

Comment:
Why didn't you create `Homework` class? It is requested in the task description, please take a closer look.